support.html

```diff
@@ -43,6 +43,13 @@ <h2><a href="{{ redHatLink }}">Red Hat</a></h2>
                 </ul>
             </p>
         </section>
+
+        <section class="col-12_md-12">
+          <h2><a href="{{ cloudRaftLink }}">CloudRaft</a></h2>
+          <p>
+              CloudRaft is an AI and Cloud Native Consulting providing support for Rook and other projects such as Kubernetes, Ceph and other open source projects.
```

Comment:
```suggestion
              CloudRaft is an AI and Cloud Native Consulting company providing support for Rook and other projects such as Kubernetes, Ceph and other open source projects.
```

Seems like a word is maybe missing here - does adding `company` here make sense?